From f63510fb086f3ef0f04855f0972d3a0dbb55fa63 Mon Sep 17 00:00:00 2001
From: Alfred Neumayer <dev.beidl@gmail.com>
Date: Fri, 21 Aug 2026 12:34:36 +0200
Subject: [PATCH 15/18] arch/arm64/configs/halium: Enable Docker & LXD

Backport of MR !6 without CONFIG_SND_SEQUENCER. It adds a field to
struct snd_rawmidi, which changes the CRCs of all snd_rawmidi_* exports.
---
 arch/arm64/configs/halium.config | 10 ++++++++++
 1 file changed, 10 insertions(+)

diff --git a/arch/arm64/configs/halium.config b/arch/arm64/configs/halium.config
index 716a91098..5f5215e89 100644
--- a/arch/arm64/configs/halium.config
+++ b/arch/arm64/configs/halium.config
@@ -30,3 +30,13 @@ CONFIG_SQUASHFS_XZ=y
 CONFIG_SQUASHFS_ZSTD=y
 # FIXME: breaks GKI ABI somehow, needed by some snaps like btop & qtcreator
 #CONFIG_CGROUP_DEVICE=y
+
+# Docker
+CONFIG_POSIX_MQUEUE=y
+CONFIG_NETFILTER_XT_MATCH_ADDRTYPE=y
+
+# LXD
+CONFIG_IP_NF_FILTER=y
+CONFIG_IP_NF_NAT=y
+CONFIG_IP6_NF_FILTER=y
+CONFIG_IP6_NF_NAT=y
-- 
2.53.0

